spec/schemas/_common.yaml

```diff
@@ -20,7 +20,6 @@ components:
       description: |-
         A duration. Units can be `nanos`, `micros`, `ms` (milliseconds), `s` (seconds), `m` (minutes), `h` (hours) and
         `d` (days). Also accepts "0" without a unit and "-1" to indicate an unspecified value.
-      x-data-type: time
```

Comment:
@Xtansia removing `x-data-type`. Do you still need this for .Net Generator? There's only 1 time schema called `Duration` and all parameters using this datatype reference this schema.

Reply:
It's currently used, but the .NET generator doesn't automatically download or re-run against the new spec so won't break anything immediately and can be easily fixed in my next generation pass. So no worries from my side about dropping it 👍